tests/api/test_endpoints.py

```diff
@@ -138,6 +138,14 @@ def test_post_clubs__failure(server):
     assert 'error' in response.json
 
 
+def test_search_clubs(server):
+# the server's test client doesn't expose a search method.
+# It does expose a GET method but that can't provide a list
+# of clubs like how the search method can.  Is there a way
+# to use the search_clubs method created in the clubs endpoint?
+# Reference:
+# https://github.com/channelcat/sanic/blob/master/docs/sanic/testing.md
```

Comment:
Just a heads up, this is a syntax error because you define a function header but give it no body, only comments which are ignored by the interpreter (see the error [here](https://travis-ci.org/ubclaunchpad/bounce/builds/413874713#L898)).

Reply:
ok, I'll add a return statement at the end of the function.  The test_client only exposes get, post, put, delete, patch, head and options methods for you to run against your application though...wondering if we can still test the search function even if it doesn't expose those methods.